docs/section23/Rule23-2.md

```diff
@@ -30,15 +30,15 @@
 ## Rule Logic:  
 
 - For each hvac system type in the building: `for baseline_system_type in baseline_hvac_system_dict:`
-  - for each hvac system in the building: `for hvac_b_id in baseline_hvac_system_dict[baseline_system_type]:`
-    - check if it is one of the eligible system types: `if any(baseline_system_type_compare(baseline_system_type, target_system_type, false) for target_system_type in target_system_types):`
-    - get the hvac system: `hvac_b = get_component_by_id(B_RMI, hvac_id)`
-      - For each fan system in HVAC system: `for fan_system_b in hvac_b.fan_systems:`
-        **Rule Assertion:**
+  - check if it is one of the eligible system types: `if any(baseline_system_type_compare(baseline_system_type, target_system_type, false) for target_system_type in target_system_types):`
+    - for each hvac system in the building: `for hvac_b_id in baseline_hvac_system_dict[baseline_system_type]:`
+      - get the hvac system: `hvac_b = match_data_element(B_RMI, HeatingVentilatingAirConditioningSystem, hvac_b_id)`
```

Comment:
python implements the match_data_element or get_component_by_id case by case.
Both ways are clear to the RCT dev team, the proposed new way might be clearer to the general public?